.github/workflows/models.yml

```
@@ -0,0 +1,30 @@
+name: Java Models
+
+on: [ workflow_dispatch, pull_request ]
+
+jobs:
+  generate:
+    runs-on: ubuntu-latest
+    name: Generate Models
+    steps:
+      - uses: actions/checkout@v3
+      - run: make models
+      - name: Set PR variables
+        id: vars
+        run: |
+          cd target/spec
+          echo ::set-output name=pr_title::"Update models"
+          echo ::set-output name=pr_body::"OpenAPI spec or templates produced new models on $(date +%d-%m-%Y) \
+            by [commit](https://github.com/Adyen/adyen-openapi/commit/$(git rev-parse HEAD))."
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v4
+        with:
+#          token: ${{ secrets.ADYEN_AUTOMATION_BOT_ACCESS_TOKEN }}
+#          committer: ${{ secrets.ADYEN_AUTOMATION_BOT_EMAIL }}
+#          author: ${{ secrets.ADYEN_AUTOMATION_BOT_EMAIL }}
```

Use @AdyenAutomationBot: 

```suggestion
          token: ${{ secrets.ADYEN_AUTOMATION_BOT_ACCESS_TOKEN }}
          committer: ${{ secrets.ADYEN_AUTOMATION_BOT_EMAIL }}
          author: ${{ secrets.ADYEN_AUTOMATION_BOT_EMAIL }}
```